pkg/clusteroperator/clusteroperator.go

```diff
@@ -0,0 +1,177 @@
+package clusteroperator
+
+import (
+	"context"
+	"time"
+
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/msg"
+	v1 "github.com/openshift/api/config/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+)
+
+const (
+	fiveScds time.Duration = 5 * time.Second
+	isTrue                 = "True"
```

Comment:
nit: IMHO this is  just redundant.
You also have a mix of comparing to isTrue on line 64 and to  "True" on line 139.